newIDE/app/src/AssetStore/InstallAsset.js

```diff
@@ -426,13 +447,15 @@ export const checkRequiredExtensionsUpdate = async ({
 
   const missingExtensionShortHeaders = filterMissingExtensions(
     gd,
-    requiredExtensionShortHeaders
+    compatibleWithIdeExtensionShortHeaders
   );
 
+  console.log(incompatibleWithIdeExtensionShortHeaders);
```

Comment:
log

Reply:
I though I removed it but I forgot to refresh my IDE.